Modules/hpe3par_snapshot.py

```diff
@@ -435,13 +447,134 @@ def restore_snapshot_online(
         snapshot_name,
         {})
 
+def create_schedule(
+        client_obj,
+        storage_system_ip,        
+        storage_system_username,
+        storage_system_password,
+        schedule_name,
+        snapshot_name,
+        base_volume_name,
+        read_only,
+        expiration_time,
+        retention_time,
+        expiration_unit,
+        retention_unit,
+        task_freq, task_freq_custom):
```

Comment:
What happens when both task_freq and task_freq_custom are provided. Which one takes priority and why?

Reply:
Have handle that in current changes

Reply:
At a time only one be taken as argument